### DISCUSSION TITLE
what is up with 22.07

### DISCUSSION THREAD

**Discussion by adsick** [helix-editor/helix#3300] [General] [answered]
I thought Helix gonna release in July but I don't see it.

**Comment by the-mikedavis** [comment, marked as answer]
We shoot for cutting releases every ~2 months but time ran a bit short in July. We're hoping to cut a 22.08 release some time this week.

#3243
https://github.com/helix-editor/website/pull/12

**Reply by David-Else** [reply]
I am on the edge of my chair for this release! :) Any update?

**Reply by Omnikar** [reply]
> I am on the edge of my chair for this release! :) Any update?

There is a Github milestone for the 22.08 release:
https://github.com/helix-editor/helix/milestone/1

**Reply by David-Else** [reply]
None of the commits in the last few days were part of the milestone... I am sure it is a 'ready when it is ready' situation, thanks to everyone working so hard to make this release amazing.
